plugins/sqs/CMakeLists.txt

```diff
@@ -34,11 +34,27 @@ if(SQS AND USE_AWS)
             set_target_properties(sqs PROPERTIES NO_SONAME 1)
         endif()
 
-        install(
-            TARGETS sqs
-            DESTINATION plugins
-            CALC_DEPS
-        )
+        if(WIN32 OR APPLE)
```

Comment:
I haven't heard of this package file conflict being an issue on macs.  I kept this logic exactly the same for any build other than linux.